README.md

```diff
@@ -14,7 +14,7 @@
 - Basic music commands.
 - Basic moderation commands.
 - Configurable, and easy to use.
-- A production-ready project, setup the bot without coding.
```

Comment:
Do not change this due to the project's major description, it cannot be changed.

Reply:
Oops, nvm.